lib/internal/main/worker_thread.js

```diff
@@ -55,6 +56,7 @@ port.on('message', (message) => {
     if (manifestSrc) {
       require('internal/process/policy').setup(manifestSrc, manifestURL);
     }
+    initializeDeprecations();
     initializeClusterIPC();
     initializeESMLoader();
     loadPreloadModules();
```

Comment:
Is there any point in trying to group these 4 or 5 calls into a single one, possibly with an options object that indicates which parts of the setup to run?

Reply:
I've tried that before but a single call seems to hide too much details - I think it should be fine to directly drop some of the calls, for example, `initializeClusterIPC` in `repl.js`. Also the order in which these executes matters to some extent and an option object would blur that.

Maybe we could group them by different modes, but I cannot think of good names for those modes off the top of my head.